### PR TITLE
chore(plugin): Add a benchmark

### PR DESCRIPTION
**Pull request by kdy1**
**Description:**

Result:

```

    Finished release [optimized] target(s) in 0.11s
es/plugin/invoke/1      time:   [125.51 us 131.64 us 139.62 us]
Found 13 outliers among 100 measurements (13.00%)
  5 (5.00%) high mild
  8 (8.00%) high severe

```

**Related issue (if exists):**

### CODE REVIEW

File: crates/swc_plugin_runner/Cargo.toml [hunk 1]
```diff
@@ -50,5 +50,11 @@ swc_ecma_parser = { version = "0.102.0", path = "../swc_ecma_parser" }
 swc_ecma_visit = { version = "0.62.0", path = "../swc_ecma_visit" }
 testing = { version = "0.19.0", path = "../testing" }
 # This allows we can run non-wasm32 target build command while some pkg select features for wasmer/js omits its transitive deps
+criterion = "0.3"
```
Comment (kwonoj) [inline crates/swc_plugin_runner/Cargo.toml:53]: comment in here (https://github.com/swc-project/swc/pull/4397/files#diff-510e57dcbb462dd6fbc8c24bd5fc0317c4d9b625b1caa117cc10b2b3d5a30d21R52) is for the `wasmer`

File: crates/swc_plugin_runner/benches/invoke.rs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        .join("rust-plugins")
+        .join("swc_internal_plugin");
+
+    {
+        let mut cmd = Command::new("cargo");
+
+        cmd.current_dir(&plugin_dir);
+        cmd.arg("build")
+            .arg("--release")
+            .arg("--target=wasm32-unknown-unknown");
```
Comment (kwonoj) [inline crates/swc_plugin_runner/benches/invoke.rs:33]: one day we may compare wasm32-wasi as well maybe?

**Review by kwonoj — APPROVED** [review]

File: crates/swc_plugin_runner/benches/invoke.rs (hunk 2 above)
Comment (kdy1) [reply crates/swc_plugin_runner/benches/invoke.rs:33]: Yeah I think we should

**Review by kdy1 — COMMENTED** [review]
swc-bump:
 - ''

**Pull request merged**